### ISSUE TITLE
conda-4.7.10 contains unsafe path: bin/activate

### ISSUE THREAD

**Issue by lucadealfaro** [conda/conda#8992]
## Current Behavior

conda update -n base -c defaults conda 

fails with the message:

CondaFileIOError: '/Users/luca/anaconda3/pkgs/conda-4.7.10-py37_0.tar.bz2'. contains unsafe path: bin/activate

### Steps to Reproduce

```
$ conda update -n base -c defaults conda
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/luca/anaconda3

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.7.10               |           py37_0         3.0 MB
    ------------------------------------------------------------
                                           Total:         3.0 MB

The following NEW packages will be INSTALLED:

  conda-package-han~ pkgs/main/osx-64::conda-package-handling-1.3.11-py37_0

The following packages will be UPDATED:

  conda                                       4.6.14-py37_0 --> 4.7.10-py37_0


Proceed ([y]/n)?


Downloading and Extracting Packages
conda-4.7.10         | 3.0 MB    | ################################################################################# | 100%

CondaFileIOError: '/Users/luca/anaconda3/pkgs/conda-4.7.10-py37_0.tar.bz2'. contains unsafe path: bin/activate
```

## Expected Behavior

Conda should be updated. 

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
     active environment : py27np16
    active env location : /Users/luca/anaconda3/envs/py27np16
            shell level : 3
       user config file : /Users/luca/.condarc
 populated config files : /Users/luca/.condarc
          conda version : 4.6.14
    conda-build version : 3.17.8
         python version : 3.7.3.final.0
       base environment : /Users/luca/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/luca/anaconda3/pkgs
                          /Users/luca/.conda/pkgs
       envs directories : /Users/luca/anaconda3/envs
                          /Users/luca/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.14 requests/2.21.0 CPython/3.7.3 Darwin/16.7.0 OSX/10.12.6
                UID:GID : 501:20
             netrc file : /Users/luca/.netrc
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /Users/luca/.condarc <==
channels:
  - defaults
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /Users/luca/anaconda3/envs/py27np16:
#
# Name                    Version                   Build  Channel
autopep8                  1.4.4                      py_0    defaults
bottle                    0.12.16                  pypi_0    pypi
ca-certificates           2019.5.15                     0    defaults
certifi                   2019.6.16                py27_0    defaults
libcxx                    4.0.1                hcfea43d_1    defaults
libcxxabi                 4.0.1                hcfea43d_1    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                h475c297_4    defaults
mysql-connector-c         6.1.11               hccea1a4_0    defaults
mysql-connector-python    2.0.4                    py27_0    defaults
mysql-python              1.2.5            py27h1de35cc_0    defaults
ncurses                   6.1                  h0a44026_1    defaults
numpy                     1.6.2                    py27_4    defaults
openssl                   1.1.1c               h1de35cc_1    defaults
pillow                    6.1.0                    pypi_0    pypi
pip                       19.1.1                   py27_0    defaults
pycodestyle               2.5.0                    py27_0    defaults
pycrypto                  2.6.1            py27h1de35cc_9    defaults
pymysql                   0.7.9                    py27_0    defaults
python                    2.7.16               h97142e2_0    defaults
readline                  7.0                  h1de35cc_5    defaults
setuptools                41.0.1                   py27_0    defaults
sqlite                    3.28.0               ha441bb4_0    defaults
tk                        8.6.8                ha441bb4_0    defaults
wheel                     0.33.2                   py27_0    defaults
yapf                      0.27.0                     py_0    defaults
zlib                      1.2.11               h1de35cc_3    defaults
```
</p></details>

**Comment by lucadealfaro** [comment]
Closing.  It was dependent on some incorrect value in some environment variable.

**Comment by msarahan** [comment]
Thanks for the follow-up! What was the offending env var, out of curiosity?

**Comment by lucadealfaro** [comment]
I wish I knew.  I killed the terminal, opened a new one to show a colleague the problem, and ... everything was working. 
A symptom in the old terminal is that I would do something like: 
$ conda activate py27np16 
to switch to an env using python 2.7 and numpy 1.6, but then when I did: 
$ python
I could see that Python was still the 3.7 version of the "base" env. 
Somehow, conda activate was not managing to activate the proper environment.
